pkg/commands/publish.ts

```diff
@@ -0,0 +1,10 @@
+import { Command } from "./command"
+
+/**
+ * @see https://redis.io/commands/publish
+ */
+export class PublishCommand extends Command<number, number> {
```

Comment:
```suggestion
export class PublishCommand<TMessage = unknown> extends Command<number, number> {
```

Reply:
Let's do it this way, so you can add arbitrary objects and they get serialized to strings internally.